### PR TITLE
add Online Stock Span

### PR DESCRIPTION
**Pull request by alexpantyukhin**
#### Description of Change

Add a guide for new LeetCode solutions.

#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [ ] Added tests and example, test must pass
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

File: leetcode/src/901.c [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    free(obj);
+}
+
+/**
+ * Your StockSpanner struct will be instantiated and called as such:
+ * StockSpanner* obj = stockSpannerCreate();
+ * int param_1 = stockSpannerNext(obj, price);
+ 
+ * stockSpannerFree(obj);
+*/
```
Comment (Panquesito7) [inline leetcode/src/901.c:67]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Comment by alexpantyukhin** [comment]
@Panquesito7 conflicts have been resolved. Please re-review

File: leetcode/src/901.c [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    free(obj->sentry);
+    free(obj);
+}
+
+/**
+ * Your StockSpanner struct will be instantiated and called as such:
+ * StockSpanner* obj = stockSpannerCreate();
+ * int param_1 = stockSpannerNext(obj, price);
+ 
+ * stockSpannerFree(obj);
```
Comment (Panquesito7) [inline leetcode/src/901.c:66]:
```suggestion
 * stockSpannerFree(obj);
 */
```

File: leetcode/src/901.c [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+ */
+}
```
Comment (Panquesito7) [inline leetcode/src/901.c:68]:
```suggestion
```

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🚀

File: leetcode/src/901.c [hunk 4]
```diff
@@ -0,0 +1,67 @@
+// Use monotonic stack.
+// Keep the stack of monotonic inxrease price and index.
```
Comment (tjgurwara99) [inline leetcode/src/901.c:2]:
```suggestion
// Keep the stack of monotonically increasing price and index.

```

**Review by tjgurwara99 — CHANGES_REQUESTED** [review]

**Review by tjgurwara99 — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**